src/java/org/apache/cassandra/cql3/PasswordObfuscator.java

```diff
@@ -18,15 +18,20 @@
 
 package org.apache.cassandra.cql3;
 
+import com.google.common.base.Optional;
+
+import org.apache.cassandra.auth.PasswordAuthenticator;
+import org.apache.cassandra.auth.RoleOptions;
+
 /**
  * Obfuscates passwords in a given string
  */
 public class PasswordObfuscator
```

Comment:
By the way it seems we have a typo. It is "obfuscatEr", I am always confused whether we need 'e' or 'o' in this word :-(

Reply:
Oh, I am not native English but googling around I can find `tOr`but not `tEr`. Isn't `tOr` correct?

Reply:
I found both and weird enough the spell checker was giving it "er". So...Merriam Webstrer - https://www.merriam-webster.com/dictionary/obfuscator

Sorry for the noise

Reply:
Ah no noise at all, always happy to learn.